spaceship/lib/spaceship/tunes/application.rb

```diff
@@ -201,8 +201,8 @@ def build_trains
 
       # The numbers of all build trains that were uploaded
       # @return [Array] An array of train version numbers
-      def all_build_train_numbers
-        client.all_build_trains(app_id: self.apple_id).fetch("trains").collect do |current|
+      def all_build_train_numbers(platform: nil)
```

Comment:
@tmm1 Good question, that's just how we handled it until now to always be in a consistent state, but I guess you're right